### PR TITLE
Fix .flo file decoding

### PR DESCRIPTION
**Pull request by NicolasHug**
I previously made a mistake in the reshaping which led to wrong values - the tests was wrong as well, so it wasn't caught. Both are now fixed.

cc @pmeier

### CODE REVIEW

File: test/test_datasets.py [hunk 1]
```diff
@@ -2095,11 +2100,16 @@ def inject_fake_data(self, tmpdir, config):
 
     @datasets_utils.test_all_configs
     def test_flow(self, config):
+        h, w = self.FLOW_H, self.FLOW_W
+        expected_flow = np.arange(3 * h * w).reshape(h, w, 3).transpose(2, 0, 1)
+        expected_flow = np.flip(expected_flow, axis=1)
+        expected_flow = expected_flow[:2, :, :]
```
Comment (NicolasHug) [inline test/test_datasets.py:2106]: This is unrelated to the fix, just a slightly more robust way to check the .pfm file decoding. I decided to disagree with myself and my previous comment

> We don't check the values because the reshaping and flipping makes it hard to figure out

**Comment by vadimkantorov** [comment]
Is there flo visualization added to pytorch utils as well?

**Comment by NicolasHug** [comment]
There isn't at the moment, but it could be interesting to add something that converts flow files into a pil/tensor image. Do you know how one usually maps a displacement value into a color map?

**Comment by vadimkantorov** [comment]
I think one first maps angle and magnitude to HSV coordinates and then converts to RGB. These utils should be included in many public optical flow repos, like RAFT

**Review by jdsgomes — APPROVED** [review]
stamping
